mealie/schema/recipe/recipe.py

```diff
@@ -102,7 +104,7 @@ def validate_tags(cats: list[Any]):  # type: ignore
     @validator("recipe_category", always=True, pre=True, allow_reuse=True)
     def validate_categories(cats: list[Any]):  # type: ignore
         if isinstance(cats, list) and cats and isinstance(cats[0], str):
-            return [RecipeCategory(id=uuid4(), name=c, slug=slugify(c)) for c in cats]
```

Comment:
I thought you were going to remove this and the other changes to the group_id requirement and fix that later down the line?  I'm concerned that generating a random group_id is going to cause errors that we haven't tested for or something else down the line.

Reply:
I have removed it from the validator. Backend tests all pass and some testing on my side seems to confirm that it doesn't have much of an impact.
It's been a while now and I'm not sure why I added the group_id to this validation function. Could have been necessary or maybe I just stumbled upon it and decided it had its place there